README.md

```diff
@@ -1,86 +1,16 @@
-401 JS --  Lab 13 express with mongo
+401 JS --  David Porter's Lab 13 Express with Mongo
 ===
 
-## Submission Instructions
-  * Work in a fork of this repository
-  * Work in a branch on your fork
-  * Write all of your code in a directory named `lab-` + `<your name>` **e.g.** `lab-duncan`
-  * Submit a pull request to this repository
-  * Submit a link to your pull request on canvas
-  * Submit a question, observation, and how long you spent on canvas  
-  
-## Learning Objectives  
-* Students will understand the purpose of a server side router 
-* Students will be able to implement a rest API using using nodes http module
+## David Porter
```

Comment:
Good reflection. I'll give you full points for the README but try to be more specific on what a user should do to successfully CREATE, READ, and DELETE.